### PR TITLE
Fix hipchat notification message capture

### LINKED ISSUE #78: Capture is not working correctly for HipChat notifications.
It seem as the {{}}-pattern substitution for capture is not working for HipChat notifications, if the message is given explicitly. E.g. in XXX is producing the first line produces

ALERT ENDED: Balance AWS: Business Partner Service Not Found Responses in Last Hour ({details}) on ad-app-tier-business-xxx-service-test-463[aws:xxx:eu-central-1]

instead of 

ALERT ENDED: Balance AWS: Business Partner Service Not Found Responses in Last Hour ({details}) on ad-app-tier-business-xxx-service-481[aws:xxx:eu-central-1]

### PR DESCRIPTION
**Pull request by mohabusama**
- Fixes #78

### CODE REVIEW

File: zmon_worker_monitor/zmon_worker/notifications/hipchat.py [hunk 1]
```diff
@@ -16,19 +16,23 @@ def send(cls, alert, *args, **kwargs):
...
 
         try:
             logger.info(
-                "Sending to: " + '{}/v2/room/{}/notification?auth_token={}'.format(url, urllib.quote(kwargs['room']),
-                                                                                   token) + " " + json.dumps(message))
+                'Sending to: ' + '{}/v2/room/{}/notification?auth_token={}'.format(url, urllib.quote(kwargs['room']),
+                                                                                   token) + ' ' + json.dumps(message))
             r = requests.post(
                 '{}/v2/room/{}/notification?auth_token={}'.format(url, urllib.quote(kwargs['room']), token),
-                data=json.dumps(message), verify=False, headers={'Content-type': 'application/json'})
```
Comment (hjacobs) [inline zmon_worker_monitor/zmon_worker/notifications/hipchat.py]: we should remove the anti-pattern `verify=False`, SSL certs should be correct

Comment (mohabusama) [inline zmon_worker_monitor/zmon_worker/notifications/hipchat.py]: Done.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**